theano/gof/op.py

```diff
@@ -933,6 +961,23 @@ def make_thunk(self, node, storage_map, compute_map, no_recycling):
         # condition: either there was no c_code, or it failed
         return self.make_py_thunk(node, storage_map, compute_map, no_recycling)
 
+    def make_node(self, *inputs):
+
+        if not hasattr(self, 'itypes'):
+            raise NotImplementedError("itypes not defined")
+
+        if not hasattr(self, 'otypes'):
```

Comment:
I mean, delete the make_node in the file compile/ops.py in the class FromFunctionOp. They can share the same code and this remove duplicate code.

thanks